Tutorials/Make_Your_Own_Intake_Datastore.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>@dougiesquire - I have a vague recollection at this step, it checks the metadata.yaml ? Is that correct, and so often runs of older models by individuals might be missing this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/440/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Nope, the <code>metadata.yaml</code> is only used when adding a datastore to the catalog </p>